src/blocks/net.rs

```diff
@@ -99,18 +102,14 @@ pub async fn run(config: toml::Value, mut api: CommonApi) -> Result<()> {
     loop {
         match NetDevice::new(device_re.as_ref()).await? {
             None => {
-                if config.hide_missing || config.hide_inactive {
-                    api.hide().await?;
-                } else {
-                    widget.set_text("×".to_string());
-                    api.set_widget(&widget).await?;
-                }
+                widget.set_format(format_missing.clone());
+                api.set_widget(&widget).await?;
             }
             Some(device) if !device.iface.is_up => {
                 if config.hide_inactive {
                     api.hide().await?;
                 } else {
-                    widget.set_text("×".to_string());
+                    widget.set_format(format_missing.clone());
```

Comment:
Since both (missing and inactive) cases use `format_missing`, I think it's safe to remove `hide_inactive` too? But I'm not sure if the distinction between these two states has its usecases. If it does, then a new format option is needed :laughing: Otherwise, a better name for `format_missing` is needed.